client/app/scripts/verifiedpixel-imagelist/module.js

```diff
@@ -1143,13 +1054,27 @@
                                 _.remove(scope.selectedList, {_id: item._id});
                             }
                         }
-                        scope.selected.preview = item;
-                        if (scope.selected.preview !== undefined) {
-                            $timeout(function(){
-                                filmstrip();
+                        var results = item.verification.results;
```

Comment:
this will fail for uploaded items without verification.  should probably use something like:

var results = (item.verification) ? item.verification.results : null;